about.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
```

Comment:
Acho melhor colocar me "me" aqui.
```suggestion
    <title>About me</title>
```

Reply:
Na real eu vi que no menu tá "about" então pode deixar assim